resources/views/people/index.blade.php

```diff
@@ -104,73 +104,14 @@
-                              fill="#F2C94C" stroke="#DCBB58"/>
-                        </svg>
-                       </span>
-                  <span class="i {{ htmldir() == 'ltr' ? 'ml3' : 'mr3' }}">{{ $contact->description }}</span>
-                 </a>
-                </li>
-               @endforeach
-
-               @foreach($unstarredContacts as $contact)
-
-                <li class="people-list-item bg-white pointer"
-                    @click="window.location.href='{{ route('people.show', $contact) }}'">
-                 <a href="{{ route('people.show', $contact) }}">
-                  @if ($contact->has_avatar)
-                   <img src="{{ $contact->getAvatarURL(110) }}" width="43">
-                  @else
-                   @if (! is_null($contact->gravatar_url))
-                    <img src="{{ $contact->gravatar_url }}" width="43">
-                   @else
-                    @if (strlen($contact->getInitials()) == 1)
-                     <div class="avatar one-letter" style="background-color: {{ $contact->getAvatarColor() }};">
-                      {{ $contact->getInitials() }}
-                     </div>
-                    @else
-                     <div class="avatar" style="background-color: {{ $contact->getAvatarColor() }};">
-                      {{ $contact->getInitials() }}
-                     </div>
-                    @endif
-                   @endif
-                  @endif
-                  <span>
-                    {{ $contact->name }}
-                  </span>
-
-                  <span class="i {{ htmldir() == 'ltr' ? 'ml3' : 'mr3' }}">{{ $contact->description }}</span>
-                 </a>
-                </li>
+                <div class="{{ \App\Helpers\LocaleHelper::getDirection() == 'ltr' ? 'fl' : 'fr' }} w-100">
```

Comment:
You can use the monica helper `htmldir()`:
```suggestion
                <div class="{{ htmldir() == 'ltr' ? 'fl' : 'fr' }} w-100">
```